nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -105,6 +108,14 @@ If this is checked, then the binaries will be built and pushed to chacra even if
           name: ARCH
           values:
             - ARCHS
+    triggers:
+      - github
+      - pollurl:
+          urls:
+            - url: https://shaman.ceph.com/api/repos/ceph/master/latest/centos/7/repo
```

Comment:
But what changes are you polling for here? I think you'll need to define ``check-content`` at least for these urls.

https://docs.openstack.org/infra/jenkins-job-builder/triggers.html#triggers.pollurl